lib/src/file_manager.dart

```diff
@@ -79,6 +81,89 @@ class _FileManagerState extends State<FileManager> {
 
   @override
   Widget build(BuildContext context) {
+    if (widget.isUsedInBuyCoverWorkFlow) {
```

Comment:
This change breaks the principle of having a reusable component. It seems to be for a specific use case.